releases/net471/KnownIssues/510901-Apps-targeting-.NET-4.6.1-who-using-.NET-Standard-libraries-might-be-broken-on-.NET-4.7.1.md

```diff
@@ -0,0 +1,18 @@
+## Symptoms
+
+Application that target .NET Framework 4.6.1 who use .NET Standard libraries could start seeing TypeLoadExceptions when running on .NET 4.7.1
+
+## Cause
+
+.NET Framework 4.7.1 adds support for .NET Standard in-box. As part of this change, a few assemblies who used to ship independently from the .NET Framework have now been added to the .NET Framework.
```

Comment:
Improvement: a few assemblies **that** used to ship independently from the .NET Framework have now been added to the .NET Framework.